### DISCUSSION TITLE
Export & Import feature [May 18]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#5742] [Discord Threads]
> 🙋‍♂️ **Jevans**

**Comment by o1lab** [comment]
> 🙋‍♂️ **Dstala | NocoDB Team**  
#### Currently it has to be done at database level. UI based action- work in progress. Expect it to be available in v0.108.0 (expected by end of this month)  
#  
> 🙋‍♂️ **Jevans**  
#### will this preserve table relations? As of now, we can dump but get a totally unlinked db that doesn't play nice in noco  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1097634743580905503)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
